snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -25,37 +25,97 @@ func snapshot(name: String, waitForLoadingIndicator: Bool = true) {
     Snapshot.snapshot(name, waitForLoadingIndicator: waitForLoadingIndicator)
 }
 
-class Snapshot: NSObject {
+func changeKeyboard(app: XCUIApplication, systemLocale: String) {
+    Snapshot.changeKeyboardToSystemLanguage(app, systemLocale: systemLocale)
+}
 
+func changeKeyboard(app: XCUIApplication) {
+    changeKeyboard(app, systemLocale: "en")
+}
+
+class Snapshot: NSObject {
+    
     class func setupSnapshot(app: XCUIApplication) {
         setLanguage(app)
         setLocale(app)
         setLaunchArguments(app)
     }
-
+    
     class func setLanguage(app: XCUIApplication) {
-        guard let prefix = pathPrefix() else {
+        guard let deviceLanguage = getDeviceLanguage(app) else {
+            print("Couldn't detect/set language...")
             return
         }
-
+        
+        app.launchArguments += ["-AppleLanguages", "(\(deviceLanguage))"]
+    }
+    
+    class func getDeviceLanguage(app: XCUIApplication) -> String? {
+        guard let prefix = pathPrefix() else {
+            return nil
+        }
+        
         let path = prefix.stringByAppendingPathComponent("language.txt")
-
+        
         do {
             let trimCharacterSet = NSCharacterSet.whitespaceAndNewlineCharacterSet()
             deviceLanguage = try NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding).stringByTrimmingCharactersInSet(trimCharacterSet) as String
-            app.launchArguments += ["-AppleLanguages", "(\(deviceLanguage))"]
+            return deviceLanguage
         } catch {
-            print("Couldn't detect/set language...")
+            print("Error loading language.txt")
         }
+        
+        return nil
     }
-
+    
+    class func changeKeyboardToSystemLanguage(app: XCUIApplication, systemLocale: String) {
+        guard let deviceLanguage = getDeviceLanguage(app) else {
+            print("Couldn't change to system language keyboard...")
+            return
+        }
+        
+        var nextKeyboardString = ""
+        
+        switch(deviceLanguage) {
```

Comment:
Is it not possible to do a more generic thing here without needing to know the localized text in the Keyboard menu? I'm making screenshots for about 50 locales and it will be a lot of work to collect the localized menu item names.

Or could we retrieve that from the Simulator somewhere?